cmd/kubeadm/app/cmd/upgrade/apply.go

```diff
@@ -194,6 +194,7 @@ func RunApply(flags *applyFlags) error {
 	// Use a prepuller implementation based on creating DaemonSets
 	// and block until all DaemonSets are ready; then we know for sure that all control plane images are cached locally
 	glog.V(1).Infof("[upgrade/apply] creating prepuller")
+	upgrade.UpgradeManifestTimeout = upgradeManifestTimeout
```

Comment:
`cmd/upgrade` imports `phases/upgrade` as a backend.
instead of modifying a package global variable in `upgrade`, why not move the constant to the `upgrade` package?

also make sure you add a comment on top of global variables / constants.

Reply:
@neolit123 Since I tried to modify the origin code  as less as I could... But this thought seems wrong... 😅